R/reportEmi.R

```diff
@@ -906,6 +1099,24 @@ reportEmi <- function(gdx, output = NULL, regionSubsetList = NULL, t = c(seq(200
                  "Emi|CO2|+|Energy (Mt CO2/yr)"))
 
   ### 2.2 Non-energy CO2 emissions ----
+  # (following q_emiAllMkt)
+  ### 2.2.1 Industrial Process Emissions ----
+
+  # if FE non-energy use available
+  if (!is.null(vm_demFENonEnergySector)) {
+    # calculate chemical process emissions from feedstocks treatment
+    EmiFac_NonEn.co2 <- collapseDim(pm_emifacNonEnergy[,,"co2"])
+    EmiProcess_Feedstocks <- pm_emifacNonEnergy[,,"co2"] * vm_demFENonEnergySector[,,getNames(EmiFac_NonEn.co2)]
+    out <- mbind(out,
+                 setNames(dimSums(vm_feedstockEmiUnknownFate, dim=3)* GtC_2_MtCO2,
+                          "Emi|CO2|Feedstocks unknown fate (Mt CO2/yr)"),
+                 setNames(dimSums(vm_incinerationEmi, dim=3)* GtC_2_MtCO2,
+                          "Emi|CO2|Plastics incineration (Mt CO2/yr)")
+                 )
```

Comment:
These two variables do not really conform to our emissions category taxonomy. Let's maybe think again how to label them and fit into the structure. Are they process or energy-related emissions?

``Emi|CO2|Plastics incineration`` could be incineration for energy purposes (IPCC 1A1) e.g. for power generation or incineration without energy recovery (IPCC 5C). Is that a correct understanding? If so, the first category we would account under ``Emi|CO2|Energy|Supply|+|Electricity/Heat``. The second we would account under ``Emi|GHG|+++|Waste``. (Or we would add a ``Emi|CO2|+|Waste``). 

Concerning the Feedstock emissions with "unknown fate", we would also need a decision of whether these emissions are energy-related (IPCC 1) or belong to some other sector. 

Maybe it is also good to check again whether the IAM community has already defined something on this (I fear not, though) e.g. by looking at the variable definitions of the NAVIGATE template.

Reply:
For the records, we fixed it with the new category `Emi|CO2|Energy|+|Waste`, under which we created: `Emi|CO2|Energy|Waste|+|Plastics Incineration` and `Emi|CO2|Energy|Waste|+|Feedstocks unknown fate`